content/rooms/v11.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Room Version 11
+type: docs
+weight: 100
+---
+
+This room version builds on [version 10](/rooms/v10) while clarifying redaction
+rules.
+
+## Client considerations
+
+Clients should no longer depend on the `creator` property of `m.room.create` events.
+
+Clients should note that the format of `m.room.redaction` events has been modified
+and look for the `redacts` key under `content` instead of a top-level event property.
+
+Clients should note that the `third_party_invite` key of `m.room.member` events
+is no longer redacted, *but* will only contain the `signed` key after redaction.
+
+Clients which implement the redaction algorithm locally should refer to the
+[redactions](#redactions) section below for a full overview.
+
+## Server implementation components
+
+{{% boxes/warning %}}
+The information contained in this section is strictly for server
+implementors. Applications which use the Client-Server API are generally
+unaffected by the intricacies contained here. The section above
+regarding client considerations is the resource that Client-Server API
+use cases should reference.
+{{% /boxes/warning %}}
+
+This room version updates the redaction algorithm and modifies how servers should
+create `m.room.create` and `m.room.redaction` events.
+
+Room version 11 is based upon room version 10 with the following considerations.
+
+### Event format
+
+The event format is unchanged by this room version. See [below](#event-format-1)
+for details on the current event format.
+
+#### Remove the `creator` property of `m.room.create` events
+
+The `m.room.create` event no longer has a `creator` property, which previously
+was always equivalent to the `sender` of the event.
+
+#### Moving the `redacts` property of `m.room.redaction` events to a `content` property
+
+The `redacts` property of `m.room.redaction` events is moved from a top-level
+event property to a property under the event `content`.
+
+For backwards-compatibility with older clients, servers should add a `redacts` property
+to the top level of `m.room.redaction` events in when serving such events over the
+Client-Server API.
+
+For improved compatibility with newer clients, servers should add a `redacts` property
+to the `content` of `m.room.redaction` events in *older* room versions when serving
+such events over the Client-Server API.
+
+### Authorization rules
```

Comment:
This is a copy of the room v10 ones with the references to `creator` of the `m.room.create` removed.

I debated about making this a separate file (and the room v10) one so that you can diff them more easily?